tests/components/snapcast/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""Tests for the Snapcast integration."""
+
+from unittest.mock import AsyncMock
+
+
+def create_mock_snapcast() -> AsyncMock:
+    """Create mock snapcast connection."""
+    mock_connection = AsyncMock()
+    mock_connection.start = AsyncMock(return_value=None)
+    return mock_connection
```

Comment:
Since there is no input, this could be turned into a fixture
```suggestion
def create_mock_snapcast() -> AsyncMock:
    """Create mock snapcast connection."""
    mock_connection = AsyncMock()
    mock_connection.start = AsyncMock(return_value=None)
    with patch("snapcast.control.create_server", return_value=mock_connection):
        yield mock_connection
```

Reply:
Ok, also moved it to conftest.py